docker-compose.yml

```
@@ -0,0 +1,24 @@
+version: "3"
+services:
+  emulator:
+    image: opentrons:latest
```

```suggestion
    image: opentrons:latest
    build .
```
this will let you use `docker-compose up --build`